### PR TITLE
Fix UI issue in stories when upgrading to v2 stylesheet

### PR DESCRIPTION
**Pull request by daiproxomhoa**
When update to v2 in file main.css we removed https://github.com/import url("https://cdn.jsdelivr.net/npm/sgds-govtech@1.3.17/css/sgds.css");
related to missing some css properties of the api table component so I will add that missing properties in styles's table component

### CODE REVIEW

**Comment by daiproxomhoa** [comment]
When update to v2 in file main.css we removed @import url("https://cdn.jsdelivr.net/npm/sgds-govtech@1.3.17/css/sgds.css");
related to missing some css properties of the api table component so I will add that missing properties in styles's table component

File: stories/storybook-common/api-table/api-table-components.tsx [hunk 1]
```diff
@@ -28,11 +28,30 @@ export const Table = ({ children }: TableProps) => {
...
+    line-height: 1.5;
+
+    code {
+        background-color: #f5f5f5;
+        color: #d0021b;
+        font-size: 0.875em;
+        font-weight: 400;
+        padding: 0.25em 0.5em;
+        white-space: pre-wrap;
+    }
```
Comment (keithtxw) [inline stories/storybook-common/api-table/api-table-components.tsx:43]: Could you help me add one for `<a>` also. It's in purple at the moment

**Review by keithtxw — CHANGES_REQUESTED** [review]

**Comment by qroll** [comment]
also spotted some styling issues on other pages:

story
- alert display story http://localhost:6006/?path=/story/modules-alert--with-custom-display

component
- nested multi select dropdown items
- nested select dropdown items

File: stories/storybook-common/api-table/api-table-components.tsx (hunk 1 above)
Comment (daiproxomhoa) [reply stories/storybook-common/api-table/api-table-components.tsx:43]: OKie

File: stories/storybook-common/api-table/api-table-components.tsx [hunk 2]
```diff
@@ -28,11 +28,36 @@ export const Table = ({ children }: TableProps) => {
 const StyledTable = styled.table`
     width: 100%;
     position: relative;
+    border-collapse: collapse;
+    border-spacing: 0;
+    line-height: 1.5;
+
+    a {
+        color: #0070f2;
```
Comment (keithtxw) [inline stories/storybook-common/api-table/api-table-components.tsx:36]: Could you use `Color.Primary`?

File: stories/storybook-common/api-table/api-table-components.tsx [hunk 3]
```diff
@@ -28,11 +28,36 @@ export const Table = ({ children }: TableProps) => {
...
+        cursor: pointer;
+        text-decoration: none;
+    }
+
+    code {
+        background-color: #f5f5f5;
+        color: #d0021b;
+        font-size: 0.875em;
```
Comment (keithtxw) [inline stories/storybook-common/api-table/api-table-components.tsx:44]: could you help me amend to use `rem`. It's our convention

**Review by keithtxw — CHANGES_REQUESTED** [review]

File: stories/storybook-common/api-table/api-table-components.tsx [hunk 4]
```diff
@@ -28,11 +28,36 @@ export const Table = ({ children }: TableProps) => {
...
+        cursor: pointer;
+        text-decoration: none;
+    }
+
+    code {
+        background-color: #f5f5f5;
+        color: #d0021b;
+        font-size: 0.875rem;
+        font-weight: 400;
+        padding: 0.25em 0.5em;
```
Comment (keithtxw) [inline stories/storybook-common/api-table/api-table-components.tsx:46]: Missed out `rem` on this

**Review by keithtxw — CHANGES_REQUESTED** [review]

**Review by keithtxw — APPROVED** [review]

**Pull request merged**